channels/ir.m3u

```diff
@@ -79,13 +79,12 @@ https://live1.presstv.com/live/hispan.m3u8
 https://live.presstv.com/live/smil:live.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="HodHodTV.ir" tvg-name="HodHod TV" tvg-country="IR" tvg-language="" tvg-logo="https://i.imgur.com/TMxCgX1.jpg" group-title="",HodHod TV
 http://51.210.199.12/hls/stream.m3u8
-#EXTINF:-1 tvg-id="iFILM.ir" tvg-name="iFILM" tvg-country="IR" tvg-language="Persian" tvg-logo="" group-title="",iFILM
-http://185.236.230.222:8000/play/a01n
-#EXTINF:-1 tvg-id="iFILMArabic.ir" tvg-name="iFILM Arabic" tvg-country="IR" tvg-language="Arabic" tvg-logo="https://i.imgur.com/RPwKZUD.jpg" group-title="Movies",iFILM Arabic (576p)
-http://163.172.209.107:8000/play/a04j/index.m3u8
```

Comment:
This link works.
![vlcsnap-2021-07-23-17h12m52s953](https://user-images.githubusercontent.com/30985701/126802963-09758f5f-862e-4191-8dfc-b663d5970fef.png)